Microsoft.Toolkit.Uwp.UI.Controls.Input/TokenizingTextBox/TokenizingTextBoxItem.Token.xaml

```diff
@@ -14,6 +14,7 @@
         <Setter Property="Visibility" Value="Visible" />
         <Setter Property="Background" Value="Transparent" />
         <Setter Property="BorderThickness" Value="0" />
+        <Setter Property="Padding" Value="8,5,8,6" />
     </Style>
```

Comment:
@michael-hawker, this isolates from the WinUI change, not sure if it isolates too much and if we should do something to try to reflex the changed is style from WinUI?